### PR TITLE
Fix the timing issue with HitagS

### PR DESCRIPTION
**Pull request by douniwan5788**
I don't have the NXP HitagS, only tested on 8211.

### CODE REVIEW

**Comment by iceman1001** [comment]
Nice!

As I look at the changes it feels like this should have been three PR's   
1.  darvin handling of mkversion
2. hitagS
3. generic lfops / em4x50 fiddling

We dislike mixed PR's since its hard to follow changes. 

If you wouldn't mind splitting up ?

**Comment by douniwan5788** [comment]
Of course, then how should I proceed? Do I have to cherry-pick to different branches?
